zokrates_core/src/flatten/mod.rs

```diff
@@ -147,23 +147,23 @@ impl<'ast> Flattener<'ast> {
                 // lhs
```

Comment:
The comment 
`check that lhs and rhs are within the right range, ie, their last two bits are zero`
could be made clear by replacing with:
`check that lhs and rhs are within the right range, i.e., their highest two highest value bits are 0. We use big-endian here, so these bits are laid out at postitions 0 and 1 by the bits directive`